extended/src/main/java/apoc/model/Model.java

```diff
@@ -60,8 +61,10 @@ public Stream<DatabaseModel> jdbc(@Name("jdbc") String urlOrKey, @Name(value = "
 
         SchemaCrawlerOptions options = SchemaCrawlerOptionsBuilder.newSchemaCrawlerOptions();
 
-        Catalog catalog = SchemaCrawlerUtility.getCatalog(getConnection(url, new LoadJdbcConfig(config)),
-                options);
+        Catalog catalog = SchemaCrawlerUtility.getCatalog(
+                (DatabaseConnectionSource) getConnection(url, new LoadJdbcConfig(config), DatabaseConnectionSource.class),
```

Comment:
Se si mette `Connection.class` invece di `DatabaseConnectionSource.class` funziona? 

In caso positivo credo si possa togliere l' `if (classType == DatabaseConnectionSource.class)` in JdbcUtil.

Altrimenti, metti un commento prima dell' `if (classType == DatabaseConnectionSource.class)`,
in cui dici che è stato messo perché ...

Reply:
No, purtroppo il metodo `getCatalog` accetta solo `DatabaseConnectionSource.class`